diff --git a/node_modules/gopd/.eslintrc b/node_modules/gopd/.eslintrc
index e2550c0..c541aa5 100644
--- a/node_modules/gopd/.eslintrc
+++ b/node_modules/gopd/.eslintrc
@@ -9,8 +9,8 @@
 		"multiline-comment-style": 0,
 		"new-cap": [2, {
 			"capIsNewExceptions": [
-				"GetIntrinsic",
-			],
-		}],
-	},
+				"GetIntrinsic"
+			]
+		}]
+	}
 }
diff --git a/node_modules/gopd/tsconfig.json b/node_modules/gopd/tsconfig.json
deleted file mode 100644
index d9a6668..0000000
--- a/node_modules/gopd/tsconfig.json
+++ /dev/null
@@ -1,9 +0,0 @@
-{
-	"extends": "@ljharb/tsconfig",
-	"compilerOptions": {
-		"target": "es2021",
-	},
-	"exclude": [
-		"coverage",
-	],
-}
